u_conn.go

```diff
@@ -162,6 +167,26 @@ func (uconn *UConn) SetSNI(sni string) {
 	}
 }
 
+// RemoveSNIExtension will cause the UConn connection client hello to not include a SNI extension
+// This only applies to non-HelloGolang ClientHelloIDs
+func (uconn *UConn) RemoveSNIExtension() error {
+	if uconn.ClientHelloID == HelloGolang {
+		return fmt.Errorf("Cannot call RemoveSNIExtension on a UConn with a HelloGolang ClientHelloID")
+	}
+	uconn.omitSNIExtension = true
+	return nil
+}
```

Comment:
This returns an `error` to prevent someone from accidentally trying to use it on a `UConn` that has a `HelloGolang` `ClientHelloID`